test/fork/Fork.t.sol

```diff
@@ -54,6 +60,8 @@ struct Deployment {
 contract ForkTest is Test {
     using stdJson for string;
 
+    address constant UNSET_ADDRESS = address(32);
```

Comment:
Probably you can add a comment about this special `address(32)` :)

Reply:
Thats a good idea!

Reply:
But why is it exactly 32?